manifests/install.pp

```diff
@@ -69,4 +69,7 @@
       ensure => absent,
     }
   }
+
+  # Install any additional packages that may be needed by the different backends
+  ensure_packages($duply_extra_packages, {'ensure' => 'present'})
```

Comment:
This is was travis is complaining about by saying

> manifests/install.pp - WARNING: top-scope variable being used without an explicit namespace on line 74

the reference should be `$duplicity::duply_extra_packages` as it is otherwise not defined in this class.